target/linux/ramips/image/mt7620.mk

```diff
@@ -179,6 +179,7 @@ define Device/dlink_dwr-116-a1
   DLINK_ROM_ID := DLK6E3803001
   DLINK_FAMILY_MEMBER := 0x6E38
   DLINK_FIRMWARE_SIZE := 0x7E0000
+  DLINK_IMAGE_OFFSET := 0x10000
```

Comment:
@CHKDSK88 Instead of adding this same line dozen of times, wouldn't it be a good idea to refactor that file little bit now? Something like this

```
define Device/dlink_dwr
  DLINK_FAMILY_MEMBER := 0x6E38
  DLINK_FIRMWARE_SIZE := 0xFE0000
  DLINK_IMAGE_OFFSET := 0x10000
  KERNEL := $(KERNEL_DTB)
  IMAGES += factory.bin
  IMAGE/sysupgrade.bin := mkdlinkfw | pad-rootfs | append-metadata
  IMAGE/factory.bin := mkdlinkfw | pad-rootfs | mkdlinkfw-factory
endef

define Device/dlink_dwr-116-a1
  $(Device/dlink_dwr)
  DTS := DWR-116-A1
  DEVICE_TITLE := D-Link DWR-116 A1/A2
  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci jboot-tools
  DLINK_ROM_ID := DLK6E3803001
  DLINK_FIRMWARE_SIZE := 0x7E0000
endef
```